crates/pet-global-virtualenvs/src/lib.rs

```diff
@@ -11,10 +12,9 @@ fn get_global_virtualenv_dirs(
     let mut venv_dirs: Vec<PathBuf> = vec![];
 
     if let Some(work_on_home) = work_on_home_env_var {
-        if let Ok(work_on_home) = fs::canonicalize(work_on_home) {
```

Comment:
Unfortunately theres a known issue in rust and this returns UNC prefix, which we do not need
https://github.com/rust-lang/rust/issues/42869